webapp/channels/src/components/post_priority/post_priority_picker_overlay.tsx

```diff
@@ -104,6 +101,7 @@ function PostPriorityPickerOverlay({
                     className={classNames({control: true, active: pickerOpen})}
                     disabled={disabled}
                     type='button'
+                    aria-label={messagePriority}
```

Comment:
This fixes the issue in the screenshot.
<img width="544" alt="Screenshot 2023-07-26 at 4 57 59 PM" src="https://github.com/mattermost/mattermost/assets/5334504/88fc7d8f-f6f5-4758-9345-9ac86aa2bc97">